exonum/src/blockchain/schema.rs

```diff
@@ -66,46 +90,51 @@ where
 
     /// Returns table that represents a map from transaction hash into raw transaction message.
     pub fn transactions(&self) -> MapIndex<&T, Hash, RawMessage> {
-        MapIndex::new("core.transactions", &self.view)
+        MapIndex::new(TRANSACTIONS, &self.view)
+    }
+
+    /// Returns table that represents a map from transaction hash into execution result.
+    pub fn transaction_results(&self) -> MapIndex<&T, Hash, TransactionResult> {
```

Comment:
Do I understand correctly that transaction results are not committed to the blockchain state? Would be great if they were - would make lying to lightweight clients about a status of a particular transaction more difficult.